### DISCUSSION TITLE
How to instrument constructor?

### DISCUSSION THREAD

**Discussion by fasheng2019** [open-telemetry/opentelemetry-java-instrumentation#4513] [Q&A]
Hi,

I want to trace whether instances of a class are created. Is there a way to specify constructor in jvm args `-Dotel.instrumentation.methods.include`? For example, `class A { public A() {...} }`, I tried `Dotel.instrumentation.methods.include="package.to.A[A]"` but it won't work. Thanks for your attention and advice!

**Comment by mateuszrzeszutek** [comment]
Hey @fasheng2019 ,
It is not currently possible to match the constructor via the `otel.instrumentation.methods.include` configuration property.
